### PR TITLE
Rename folder crml/cennzx-spot -> crml/cennzx

### PR DESCRIPTION
**Pull request by jordy25519**
Remove unused ScaleDownFactor var
Fix doc comments on cennzx buy/sell asset

### CODE REVIEW

File: crml/cennzx/src/lib.rs [hunk 1]
```diff
@@ -106,13 +106,13 @@ decl_module! {
...
-		/// User specifies an exact `buy_amount` and a `maximum_sell` amount.
+		/// Caller specifies an exact `buy_amount` and a `maximum_sell` amount to pay.
 		///
-		/// `recipient` - Account to receive `buy_amount`, defaults to `origin` if None
+		/// `recipient` - Account to receive assets, defaults to `origin` if None
 		/// `asset_to_sell` - asset ID to sell
 		/// `asset_to_buy` - asset ID to buy
-		/// `buy_amount` - The amount `asset_to_buy` to purchase
-		/// `maximum_sell` - Maximum `asset_to_sell` to pay
+		/// `buy_amount` - The amount of `asset_to_buy` caller should receive
```
Comment (KarishmaBothara) [inline crml/cennzx/src/lib.rs]: caller / recipient should receive?

**Review by KarishmaBothara — APPROVED** [review]
lgtm

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -112,7 +112,6 @@ parameter_types! {
 	pub const AvailableBlockRatio: Perbill = Perbill::from_percent(75);
 	pub const MaximumBlockLength: u32 = 5 * 1024 * 1024;
 	pub const Version: RuntimeVersion = VERSION;
-	pub const ScaleDownFactor: Balance = 1_000_000_000_000;
```
Comment (aliXsed) [inline runtime/src/lib.rs:115]: Possibly could be better to be in a different PR
